### ISSUE TITLE
[TextInputLayout][Accessibility] Focus is lost after activating the clear button

### ISSUE THREAD

**Issue by tillygajanayake** [material-components/material-components-android#1081]
**Description:** The focus(Touch target) is lost after activating the clear button when the TalkBack is on in a TextInputLayout. Please follow the steps below.
1. Enter some text to the text input field.
2. Enable Talkback.
3. Select the clear button in the text input field.
4. Double-tap to activate.

Please observe the sequence bellow.
<img src="https://user-images.githubusercontent.com/61483610/75739915-51db7c80-5d5a-11ea-9334-7fc20349b265.gif" width=300>

**Expected behavior:** The focus should be transferred to the text input field after activating the clear button when the TalkBack is on in a TextInputLayout.

**Android API version:** Android API version 29

**Material Library version:** 1.1.0

**Device:** Samsung Galaxy S10+ / Pixel 3

**Comment by leticiarossi** [comment]
It's not losing focus, the focus is going back to the edit text but since there's no text, nothing in special is announced. On my end it announces that the keyboard is open. If there was a placeholder text, for example, that'd be announced.
